docs/controlpanel.rst

```diff
@@ -0,0 +1,7 @@
+Control Panels
+===============
+
+Status-ControlPanel:
+---------------------
+The status control panel will check subprocesses that Castle-CMS needs. This control panel checks that docsplit is
```

Comment:
- please use the correct spelling of "CastleCMS"
- break out each of these tests into point form
- add elasticsearch
- rather than say that it checks that Celery is working properly, it would be more correct to say that CastleCMS is able to connect to a Celery worker